core/apigateway-profiling/Dockerfile

```diff
@@ -0,0 +1,143 @@
+# apigateway
+#
+# VERSION               1.9.7.3
+#
+# From https://hub.docker.com/_/alpine/
+#
+FROM ubuntu:xenial
```

Comment:
update the comment above which says `alpine` ?

Reply:
Whoops.  Will correct.